diff --git a/index.js b/index.js
index 0d0ba443385ca92344ed22fe8d77bf8c617273ce..93b680422d96ce49e7623c60590e3c100572e394 100644
--- a/index.js
+++ b/index.js
@@ -189,7 +189,8 @@ export default function ({ external = [], edge, split } = {}) {
 					platform: 'browser',
 					format: 'esm',
 					external,
-					sourcemap: 'linked'
+					sourcemap: 'linked',
+					banner: { js: 'var global = globalThis;' } 
 				});
 
 				write(